### PR TITLE
NatGeo transition

### PR DESCRIPTION
**Pull request by tbaranes**
Related to #155 

This PR add the `NatGeo` animations:
- Navigation: present / dismiss, push / pop
- Two directions: Left / Right (don't think it's really useful to have Top / Bottom)
- No custom parameters  (could have the ratio, but after playing with it, it rarely render well)
- App example updated

👯

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -17,6 +17,7 @@ All notable changes to this project will be documented in this file.
 - Add `ExplodeAnimator` to support Explode transition animation. It supports parameters `Explode(xFactor, minAngle, maxAngle)`, if no specified, the default values are `Explode(10, -10, 10)`. [#155](https://github.com/JakeLin/IBAnimatable/issues/155)
 - Add `FoldAnimator` to support Fold transition animation. It supports parameters `Explode(direction, nbFolds)`, if no specified, the default values are `Fold(Left, 2)`. [#155](https://github.com/JakeLin/IBAnimatable/issues/155)
 - Add `PortalAnimator` to support Portal transition animation. It supports parameters `Portal(direction, zoomScale)`, if no specified, the default values are `Portal(Forward, 0.8)`. [#155](https://github.com/JakeLin/IBAnimatable/issues/155)
+- Add `NatGeoAnimator` to support NatGeo transition animation. It supports only a direction `Portal(direction)`, if no specified, the default values are `Portal(Left)`. [#155](https://github.com/JakeLin/IBAnimatable/issues/155)
```
Comment (JakeLin) [inline CHANGELOG.md]: I think there is a typo, <del> Portal</del> should be `NatGeo(direction)` and `NatGeo(Left)`

File: IBAnimatable/PresentNatGeoWithDismissInteractionSegue.swift [hunk 2]
```diff
@@ -0,0 +1,14 @@
+//
+//  Created by Tom Baranes on 24/04/16.
+//  Copyright © 2016 Jake Lin. All rights reserved.
+//
+
+import UIKit
+
+public class PresentNatGeoWithDismissInteractionSegue: UIStoryboardSegue {
+  public override func perform() {
+    destinationViewController.transitioningDelegate = PresenterManager.sharedManager().retrievePresenter(.NatGeo(direction: .Forward), interactiveGestureType: .Default)
```
Comment (JakeLin) [inline IBAnimatable/PresentNatGeoWithDismissInteractionSegue.swift]: should `. Forward` replace with `. Left`?

File: IBAnimatable/NatGeoAnimator.swift [hunk 3]
```diff
@@ -0,0 +1,186 @@
...
+    containerView.addSubview(toView)
+    containerView.addSubview(fromView)
+    if fromDirection == .Left {
+      executeLeftAnimation(transitionContext, containerView: containerView, fromView: fromView, toView: toView)
+    } else {
+      executeRightAnimations(transitionContext, containerView: containerView, fromView: fromView, toView: toView)
+    }
+  }
+  
+  private func animationDidFinish(transitionContext: UIViewControllerContextTransitioning, containerView: UIView, fromView: UIView, toView: UIView) {
```
Comment (JakeLin) [inline IBAnimatable/NatGeoAnimator.swift]: May be better to put this method inside the Helper extension.

**Comment by JakeLin** [comment]
👍 great one, thanks @tbaranes

**Comment by tbaranes** [comment]
Thanks for your review, just commit the changes 🎉

**Comment by JakeLin** [comment]
@tbaranes thanks, merge it when you free it is ready.

**Pull request merged**